<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RegeltestSetup_QuestionGroup</class>
    <widget class="QWidget" name="RegeltestSetup_QuestionGroup">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>88</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
       <widget class="QLabel" name="label_question_group">
           <property name="text">
               <string>01 - Das Spielfeld</string>
           </property>
       </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="left_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Textfragen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_textquestion">
        <property name="suffix">
         <string> out of 32</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Multiple choice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_mchoice">
        <property name="suffix">
         <string> out of 8</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="right_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
